samples/change-notification/csharp/ChangeNotification/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "ConnectionName": "",
+  "BaseUrl": "",
+  "UserId": ""
```

Comment:
Why we are putting `userId` in configuration?

Reply:
It is required to subscribe the user presence on the basis of userId.

Reply:
Can we read for from Graph API or bot context? I believe you are setting up the same user id in config who has installed this app

Reply:
Ya Trinetra, We can get it form bot context.